tests/test_channels/test_partial_transpose.py

```diff
@@ -13,19 +13,19 @@ def test_partial_transpose_bipartite():
     rho = np.arange(16).reshape(4, 4)
 
     # Partial transpose of first subsystem:
-    res = partial_transpose(rho, [1], [2, 2])
+    res = partial_transpose(rho, [0], [2, 2])
```

Comment:
We should also ensure that we include a test to cover the case when `sys` is provided as an `int` (instead of a list with a single `int` inside of it). Something like:

```python
def test_partial_transpose_sys_int():
    """Partial transpose `sys` argument is provided as `int`."""
    test_input_mat = np.arange(1, 17).reshape(4, 4)

    expected_res = np.array([[1, 2, 9, 10], [5, 6, 13, 14], [3, 4, 11, 12], [7, 8, 15, 16]])

    res = partial_transpose(test_input_mat, sys=0)

    bool_mat = np.isclose(res, expected_res)
    np.testing.assert_equal(np.all(bool_mat), True)
```